src/index.d.ts

```diff
@@ -1,5 +1,5 @@
 declare namespace uvu {
-	type Callback<T> = (context: T) => Promise<void> | void;
+	type Callback<T> = (context: T, breadcrumbs?: [suiteName?: string, testName?: string, ...crumbs: string[]]) => Promise<void> | void;
```

Comment:
```suggestion
	type Callback<T> = (context: T, breadcrumbs?: string[]) => Promise<void> | void;
```

could be an alternative to be a tad bit more generic—but as it stands now, there are only 2 crumbs so we could name them.

Reply:
I'd rather just be generic 👍